ch_backup/backup/metadata/table_metadata.py

```diff
@@ -53,6 +55,24 @@ def get_parts(self, *, excluded_parts: Set[str] = None) -> List[PartMetadata]:
                     PartMetadata.load(self.database, self.name, part_name, raw_metadata)
                 )
 
+        if self.engine == REPLACING_MERGE_TREE:
```

Comment:
What about ReplicatedReplacingMergeTree and table engines like CollapsingMergeTree? Wouldn't it better to perform the code below unconditionally?

Reply:
Good idea.